openupgradelib/openupgrade.py

```diff
@@ -582,6 +583,65 @@ def rename_columns(cr, column_spec):
                 )
 
 
+def _rename_field_on_dashboard(env, model, old_field, new_field,
+                               dashboard_view_data):
```

Comment:
This function needs a docstring - it is unclear what dashboard_view_data is.

However from an API perspective, my preference would be that this function was

```def _rename_field_on_dashboard(env, field_spec):
    if version_info[0] < 10:
        return
    # Variable used to change field which present in ir.ui.view.custom
    # like the one used on dashboard module
    dashboard_view_data = env["ir.ui.view.custom"].search([])
```

Reply:
`dashboard_view_data` must disappear as argument and do the search for relevant data containing `old_field`.

Reply:
Yes sorry, bad markdown by me, I did write that.